cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -39,27 +35,20 @@
     {% if video_url.find( 'autoplay=' ) == -1 -%}
         {% set video_url = _buildParam( video_url, 'autoplay=1' )  %}
     {% endif %}
-    {% if video_player == 'youtube' -%}
-        {% set video_id =  video_url.split('?')[0].split('/') | last %}
-        {% if video_url.find( 'enablejsapi=' ) == -1 -%}
-            {% set video_url = _buildParam( video_url, 'enablejsapi=1' )  %}
-        {% endif -%}
-        {% if video_url.find( 'origin=' ) == -1 -%}
-            {% set video_url = _buildParam(
-                 video_url,
-                 'origin=' ~ request.scheme ~ '://' ~ request.META['HTTP_HOST']
-            ) %}
-        {% endif %}
-    {% elif video_player == 'ustream' %}
-        {% if video_url.find( 'html5ui' ) == -1 -%}
-            {% set video_url = _buildParam( video_url, 'html5ui=1' )  %}
-        {% endif %}
-        {% set image_loaded_class = 'video-player_image-loaded' %}
+    {% set video_id =  video_url.split('?')[0].split('/') | last %}
+    {% if video_url.find( 'enablejsapi=' ) == -1 -%}
+        {% set video_url = _buildParam( video_url, 'enablejsapi=1' )  %}
+    {% endif -%}
+    {% if video_url.find( 'origin=' ) == -1 -%}
+        {% set video_url = _buildParam(
+             video_url,
+             'origin=' ~ request.scheme ~ '://' ~ request.META['HTTP_HOST']
+        ) %}
     {% endif %}
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}
+                video-player__youtube
```

Comment:
Do we steen need the `youtube` modifier at all?

Reply:
yeah, just for now, until I can cleanup the JS code.